src/bin/commands/clean.rs

```diff
@@ -0,0 +1,74 @@
+use std::fs;
+
+use super::containers::*;
+use super::images::*;
+use clap::Args;
+
+#[derive(Args, Debug)]
+pub struct Clean {
+    /// Provide verbose diagnostic output.
+    #[clap(short, long)]
+    pub verbose: bool,
+    /// Force removal of images.
+    #[clap(short, long)]
+    pub force: bool,
+    /// Remove local (development) images.
+    #[clap(short, long)]
+    pub local: bool,
+    /// Remove images. Default is a dry run.
+    #[clap(short, long)]
+    pub execute: bool,
```

Comment:
not providing `--execute` should probably display a note that to execute, pass --execute

Reply:
same should be done in `images` subcommand